### DISCUSSION TITLE
PA1 grading criteria

### DISCUSSION THREAD

**Discussion by livelykitten** [ANLAB-KAIST/KENSv3#140] [Project Q&A 1. Application Layer (Socket)] [answered]
PA1 document says that the test score on the test cases provided will be our grade on PA1. There will be no other test cases that we don't know. 

Could you confirm that I understood it correct?

Thank you

PA1은 echo를 실행시켰을 때 나오는 테스트 케이스 15개 전부 통과하면 만점을 받는 게 맞는지 확인해주실 수 있나요?

감사합니다

**Comment by Pusnow** [comment, marked as answer]
If you pass all provided test cases, you will get full credit.
